spec.md

```diff
@@ -51,13 +51,14 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 
 The high-level components of the spec include:
 
-* [Image Manifest](manifest.md) - Describes the components that make up a container image.
-* [Image Index](image-index.md) - An index of annotated image manifests distributed as a complete unit.
-* [Image Layout](image-layout.md) - The filesystem layout representing the contents of an image
+* [Image Manifest](manifest.md) - a document describing the components that make up a container image
+* [Image Index](image-index.md) - an index of annotated image manifests distributed as a complete unit
```

Comment:
“an index of annotated image manifests” → “an annotated index of image manifests”?  The index has annotated descriptors pointing at image manifests; whether the manifests themselves contain annotations doesn't really matter here.

And I'd rather drop “complete unit”, since the presence of an index is really about indexing multiple things, and not about whether the DAGs referenced from the index are present in the same CAS engine.